docs/drill4net/drill4net-scanner-parameters.mdx

```diff
@@ -3,6 +3,21 @@ id: drill4net-scanner-parameters
 title: Drill4Net Scanner Parameters
 ---
 
+## Example
+```
+Drill4Net.Scanner.exe <path that contains executable file> --target <path to executable file> --exclude "[coverlet.*]Coverlet.Core.Coverage"
+```
+Drill4Net goes a step in the other direction by also letting you explicitly set what can be included using the `--include` option.  
```

Comment:
`Drill4Net allows to explicitly set what must be included using the `--include` option. ` - less words to read -> easier to understand